source/voice_control/apple.markdown

```diff
@@ -2,30 +2,56 @@
 title: "Assist on Apple devices"
 ---
 
-Assist is integrated via [Home Assistant Companion](https://apps.apple.com/us/app/home-assistant/id1099568401) into Apple devices using the [Shortcuts app](https://support.apple.com/guide/shortcuts/welcome/ios). Assist can be used in the following ways on Apple devices:
+Assist can be used on Apple devices via [Home Assistant Companion App](https://apps.apple.com/us/app/home-assistant/id1099568401).
 
-- Activated by Siri by saying the shortcut name "Hey Siri, Assist"
-- Added to your iPhone home screen
-- Pinned to your Mac menu bar
-- Activated via a keyboard shortcut on your Mac
-- Added as an Apple Watch complication
+## Assist on iPhones
 
-<lite-youtube videoid="sQ7X7jz1SrA" videotitle="Assist on Apple HomePod"></lite-youtube>
+Assist is available on iPhones and tablets.
 
-## Installation
+### Prerequisites
 
-To get started make sure you're running the latest version of the [Home Assistant Companion app](https://apps.apple.com/us/app/home-assistant/id1099568401?itsct=apps_box_badge&itscg=30200) and have updated your Apple devices. Then download the two shortcuts and open them to import them:
+- [Home Assistant Companion App](https://companion.home-assistant.io/docs/getting_started/) installed
+- Have an Assistant set up: either [cloud](https://www.home-assistant.io/voice_control/voice_remote_cloud_assistant/) (recommended, more performant) or [local](https://www.home-assistant.io/voice_control/voice_remote_local_assistant/).
+- The devices you want to control with Assist are [exposed to Assist](/voice_control/voice_remote_expose_devices/)
 
-- [Assist shortcut](https://www.icloud.com/shortcuts/4172210db1d74599bf9ae4f9e1e3d6ed)
-- [Assist Button shortcut](https://www.icloud.com/shortcuts/73ebb68ca1a24ae1811b7d6971203334)
+### Starting Assist in Home Assistant
 
-![Expanded Shortcuts Action](/images/assist/ios-expanded-shortcut-action.jpeg)
+1. On your phone, open Home Assistant.
+2. In the top-right corner, select the three-dots menu and select **Assist**.
+3. [Give a command](/voice_control/custom_sentences/).
 
-There are two versions of the shortcut. The normal shortcut works better with Siri while the Button shortcut has the ability to use speech-to-text when triggered outside of Siri and can customize the input language (instead of relying on the OS language)
+### Starting Assist on your iPhone using a shortcut
 
-[_Last updated: Jan 26, 2023_](#changelog)
+This section was written using an iOS 18. Depending on your model and version, the exact steps may vary.
 
-### Adjusting the language
+To use Home Assistant Assist as your voice assistant, follow these steps:
+
+1. Create a shortcut to **Assist in App**
```

Comment:
```suggestion
1. Create a shortcut to **Assist in app**
```

Reply:
There are currently duplicates there and the correct is the one with "app" lowercased, I am opening a PR adding "Legacy" to the old one